boba.tokenlist.json

```diff
@@ -41,6 +65,22 @@
       "decimals": 18,
       "logoURI": "https://raw.githubusercontent.com/bobanetwork/token-list/main/assets/ethereum.svg"
     },
```

Comment:
lets add L2_ETH for goerli as well! at 0x4200000000000000000000000000000000000006